apps/fabric-website/src/components/Site/Site.tsx

```diff
@@ -140,7 +139,6 @@ export class Site<TPlatforms extends string = string> extends React.Component<IS
   public render() {
     const { platform, isContentFullBleed } = this.state;
     const { children, siteDefinition } = this.props;
-    const { customizations } = siteDefinition;
```

Comment:
We still want to leave customizations on the Site component since it's intended to be consumed. Anyone should be able to add site wide customizations from SiteDefinition.

Reply:
To expand: The HIG site doesn't have it's own Site component or createSite factory. It imports createSite from the fabric-website package and provides it's own Platforms interface, SiteDefinition, and page components. It even directly imports the controls and styles definition.

Reply:
But do we have any reason for using customizations in the HIG now that everything is fluent by default?

Reply:
The HIG doesn't have that use case currently, but I believe it would be best practice to continue supporting customizations for the Site component, since anyone can use it to create a site.

Reply:
I guess that makes sense, though I question in practice whether anyone would actually use these components to make their own site due to the lack of documentation and various parts of the design/implementation being tightly tied to our scenario.